Trailers in the "verbose" section

If you have a trailer like things in the body of the message, it picks them up as trailers.

Example error

```
Move to specdown
# Short (50 chars or less) summary of changes
#
# More detailed explanatory text, if necessary.  Wrap it to
# about 72 characters or so.  In some contexts, the first
# line is treated as the subject of an email and the rest of
# the text as the body.  The blank line separating the
# summary from the body is critical (unless you omit the body
# entirely); tools like rebase can get confused if you run
# the two together.
#
# Further paragraphs come after blank lines.
#
#   - Bullet points are okay, too
#
#   - Typically a hyphen or asterisk is used for the bullet,
#     preceded by a single space, with blank lines in
#     between, but conventions vary here

# Bitte geben Sie eine Commit-Beschreibung für Ihre Änderungen ein. Zeilen,
# die mit '#' beginnen, werden ignoriert, und eine leere Beschreibung
# bricht den Commit ab.
#
# Auf Branch specdown
# Zum Commit vorgemerkte Änderungen:
#	geändert:       Makefile
#	gelöscht:       bin/mdtest
#	geändert:       docs/lints/index.md
#
# ------------------------ >8 ------------------------
# Ändern oder entfernen Sie nicht die obige Zeile.
# Alles unterhalb von ihr wird ignoriert.
diff --git a/Makefile b/Makefile
index 0d3fc98..38a2784 100644
--- a/Makefile
+++ b/Makefile
@@ -13,10 +13,10 @@ show-help:
 test:
 	unset GIT_MIT_AUTHORS_EXEC && RUST_BACKTRACE=1 cargo test --locked
 
-.PHONY: mdtest
-## Test the markdown in the usage directory
-mdtest: build
-	./bin/mdtest ./usage/**/*.md
+.PHONY: specdown
+## Test the markdown in the docs directory
+specdown: build
+	./bin/specdown ./docs/**.md ./docs/**/*.md
 
 .PHONY: smoke-test
 ## Run a smoke test and see if the app runs
diff --git a/bin/mdtest b/bin/mdtest
deleted file mode 100755
index c5345e6..0000000
--- a/bin/mdtest
+++ /dev/null
@@ -1,39 +0,0 @@
-#!/usr/bin/env bash
-
-set -euo pipefail
-
-# markdown bash execute
-function mbe() {
-    if [ -f "$1" ]; then
-        MARKDOWN_PATH="$(
-            cd "$(dirname "$1")"
-            pwd -P
-        )/$(basename "$1")"
-
-        (
-            unset GNUPGHOME
-            unset EDITOR
-            unset VISUAL
-            unset GIT_MIT_AUTHORS_EXEC
-            unset GIT_MIT_AUTHORS_CONFIG
-            PATH="$PWD/target/release/:$PATH"
-
-            cd "$(mktemp -d)"
-            # shellcheck disable=SC2016
-            sed -n '/``` bash/,/```/p' <"$MARKDOWN_PATH" | # get the bash code blocks
-                sed 's/``` bash//g' |                      #  remove the ```bash
-                sed 's/```//g' |                           # remove the trailing ```
-                sed '/^$/d' |                              # remove empty lines
-                /usr/bin/env bash                          # execute the command
-        )
-    else
-        echo "$1 is not valid"
-        exit 1
-    fi
-}
-### Credit: https://gist.github.com/gmolveau/67d10dfaea1fdd729865b2f8d46f7488
-for ARG in "${@:1}"; do
-    echo "Trying: $ARG"
-    echo
-    mbe "$ARG"
-done
diff --git a/docs/lints/index.md b/docs/lints/index.md
index 8cb48db..1eb9ccb 100644
--- a/docs/lints/index.md
+++ b/docs/lints/index.md
@@ -68,13 +68,13 @@ git commit --message="$(cat message)"
 
 ```text,verify(script_name="1", stream=stderr)
 Demonstration Commit Message
- 
+
 This is a commit message that has trailers and is invalid
 
-Signed-off-by: Someone Else <someone@example.com>
-Signed-off-by: Someone Else <someone@example.com>
 Co-authored-by: Billie Thompson <billie@example.com>
 Co-authored-by: Billie Thompson <billie@example.com>
+Signed-off-by: Someone Else <someone@example.com>
+Signed-off-by: Someone Else <someone@example.com>
 
 
 ---
@@ -82,6 +82,7 @@ Co-authored-by: Billie Thompson <billie@example.com>
 Your commit message has duplicated trailers
 
 You can fix this by deleting the duplicated "Signed-off-by", "Co-authored-by" fields
+
 ```
 
 #### Git Manual Style


---

Your commit message has duplicated trailers

You can fix this by deleting the duplicated "Co-authored-by" field
```

--- a/mit-commit-message-lints/src/lints/lib/duplicate_trailers.rs
+++ b/mit-commit-message-lints/src/lints/lib/duplicate_trailers.rs
@@ -148,6 +148,46 @@ mod tests_has_duplicated_trailers {
             )),
         );
     }
+
+    #[test]
+    fn trailer_like_duplicates_in_the_scissors_section() {
+        test_lint_duplicated_trailers(
+            indoc!(
+                "
+                Move to specdown
+                # Bitte geben Sie eine Commit-Beschreibung fur Ihre Anderungen ein. Zeilen,
+                # die mit '#' beginnen, werden ignoriert, und eine leere Beschreibung
+
+                # ------------------------ >8 ------------------------
+                # Andern oder entfernen Sie nicht die obige Zeile.
+                # Alles unterhalb von ihr wird ignoriert.
+                diff --git a/Makefile b/Makefile
+                index 0d3fc98..38a2784 100644
+                --- a/Makefile
+                +++ b/Makefile
+                +
+                 This is a commit message that has trailers and is invalid
+
+                -Signed-off-by: Someone Else <someone@example.com>
+                -Signed-off-by: Someone Else <someone@example.com>
+                 Co-authored-by: Billie Thompson <billie@example.com>
+                 Co-authored-by: Billie Thompson <billie@example.com>
+                +Signed-off-by: Someone Else <someone@example.com>
+                +Signed-off-by: Someone Else <someone@example.com>
+
+
+                 ---
+                @@ -82,6 +82,7 @@ Co-authored-by: Billie Thompson <billie@example.com>
+                 Your commit message has duplicated trailers
+
+                 You can fix this by deleting the duplicated \"Signed-off-by\", \"Co-authored-by\" fields
+                +
+                "
+            )
+            .into(),
+            &None,
+        );
+    }
     #[test]
     fn other_trailers() {
         test_lint_duplicated_trailers(

